scrape/scrape.go

```diff
@@ -61,6 +62,32 @@ var (
 		},
 		[]string{"interval"},
 	)
+	targetScrapePools = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "prometheus_target_scrape_pools_total",
+			Help: "Total number of scrape pool creation atttempts.",
+		},
+	)
+	targetScrapePoolsFailed = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "prometheus_target_scrape_pools_failed_total",
+			Help: "Total number of scrape pool creations that failed.",
+		},
+	)
+	targetScrapePoolReloads = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "prometheus_target_scrape_pool_reloads_total",
+			Help: "Total number of reloads on a scrape pool.",
+		},
+		[]string{"scrape_job"},
```

Comment:
There could be a lot of scrape jobs, I'd not have this label.

Reply:
Fine for me. Note that `prometheus_target_sync_length_seconds` and `prometheus_target_scrape_pool_sync_total` already include that label.